config/api_server_deployment.yml

```diff
@@ -87,10 +87,10 @@ spec:
           resources:
             requests:
               cpu: 500m
-              memory: 15Mi
+              memory: 100Mi
             limits:
-              cpu: 15000m
-              memory: 150Mi
+              cpu: 1500m
+              memory: 1Gi
```

Comment:
I think this CPU limit must have been a typo before. I've adjusted it to a more reasonable multi-core limit so that it can get some concurrency if it needs to. I've also adjusted the memory limit to 1Gi.

The registry-buddy converts a zip file containing package source code into a tar file/single-layer OCI image and sends it to a container registry. For apps with large individual files (e.g. a Java app with jars) we've seen memory surge past the 150Mi this had originally and the `cf-api-servers` Pod would get OOMKilled.  I'm considering `2Gi` to account for the default max package size, though I do not think it should actually use that much memory based on my understanding of the code:

https://github.com/cloudfoundry/capi-k8s-release/blob/5878960c0ad229fbd5832c728e3addf686e2f18f/src/registry-buddy/archive/archive.go#L119

Thoughts on this?